### DISCUSSION TITLE
A problem with downloading: https://www.rtve.es/play/videos/cine-internacional/spectre/7024872/

### DISCUSSION THREAD

**Discussion by Samuelherreroedo** [aclap-dev/video-downloadhelper#583] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://www.rtve.es/play/videos/cine-internacional/spectre/7024872/",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [dash @ 000001892a936600] Error when loading first fragment of playlist\r\n[in#0 @ 000001892a936440] Error opening input: Server returned 404 Not Found\r\nError opening input file https://ztnr.rtve.es/ztnr/7024872.mpd.\r\nError opening input files: Server returned 404 Not Found\r\n",
    "id": "coapp_failure_67fe13cb-cf57-46e2-a201-eba052b8af3b",
    "downloadable_id": "downloadable_967915496901274"
  }
}

### Addon & CoApp version and details

I try to download a video from RTVE.es and I cannot.

**Comment by paulrouget** [comment]
Please no caps: https://github.com/aclap-dev/video-downloadhelper/wiki/Code-Of-Conduct

Triage notes: this is a header failure most likely.

**Reply by Samuelherreroedo** [reply]
sorry for the cabs.
rtve means: radio television española. in spain we put always the first letter in cabs.

**Comment by Samuelherreroedo** [comment]
please, if you can arrange the failure for the future.
best regards
